src/tools/genpolicy/rules.rego

```diff
@@ -549,6 +549,23 @@ allow_linux_devices(p_devices, i_devices) {
     print("allow_linux_devices: true")
 }
 
+allow_linux_sysctl(p_linux, i_linux) {
+    print("allow_linux_sysctl 1: start")
+    not i_linux["Sysctl"]
```

Comment:
@burgerdev , would `i_linux.Sysctl` work here too, instead of `i_linux["Sysctl"]`?

If yes, I would prefer `i_linux.Sysctl` - for consistency with other rules, and to avoid adding a few more characters to our already long Policy.

Reply:
After reading the spec more carefully, I realize that object field access with square brackets and dot is actually interchangeable.